docs/wiki/dev-guides/GitHub-Conventions.md

```diff
@@ -34,10 +34,10 @@ docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/
 1. Fetch the pull request ref and assign it to a local branch name.
 
    ```bash
-   git fetch upstream pull/<ID>/HEAD/:LOCAL_BRANCH_NAME
+   git fetch <upstream> pull/<ID>/head:LOCAL_BRANCH_NAME
```

Comment:
lets change to `<remote>` as per convo, and lets just fix up the language on that other page while we're at it since it seems to insinuate that we'll be using `origin` for all the docs